frontend/packages/dev-console/locales/en/dev-console.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Remove this file since you're using `devconsole` for i18n strings now.

Reply:
Removed.